llvm-general/src/LLVM/General/Internal/FFI/TargetC.cpp

```diff
@@ -338,22 +316,4 @@ void LLVM_General_InitializeAllTargets() {
   // None of the other components are bound yet
 }
 
-LLVMBool LLVM_General_TargetMachineEmit(
```

Comment:
Please put the previous structure back. The point of exposing the interface which writes to streams is to eventually reduce the number of copies of the output code when doing stuff like JIT or streaming the result over the network.